CONTRIBUTING.md

```diff
@@ -87,28 +87,22 @@ read `src/jou_compiler.h` and have a quick look at `src/util.h`.
 
 ## Tests
 
-**Note: Currently tests do not work on windows.**
-
 GitHub Actions runs all tests when you make a pull request,
 so you don't need to run tests locally if you only intend to fix a couple small things.
 That said, test-driven development works very well for developing compilers.
 There should be a test (or a TODO comment about adding a test)
 for every feature and for every compiler error/warning.
 
-Running tests:
+Running tests (if on Windows, use Git Bash):
 
 ```
-$ make test         # Run all tests quickly. Good for local development.
-$ make valgrind     # Run some of the tests with valgrind.
-$ make fulltest     # Very slow. Includes the other two. Runs in CI.
+$ ./runtests.sh
 ```
 
-You need valgrind (e.g. `sudo apt install valgrind`) for `make valgrind` and `make fulltest`.
-
-Each of these commands:
-- compiles the Jou compiler if you have changed something in `src/` since the last time it was compiled
-- runs all Jou files in `examples/` and `tests/` (`make valgrind` only runs some files, see below)
-- ensures that the Jou files output what is expected.
+This command does a few things:
+- It compiles the Jou compiler if you have changed something in `src/` since the last time it was compiled
```

Comment:
```suggestion
- If not on Windows, it compiles the Jou compiler if you have changed something in `src/` since the last time it was compiled. (On Windows you need to use the build button in CodeBlocks to compile.)
```